Sources/AudioKit/Sequencing/Apple Sequencer/AppleSequencer.swift

```diff
@@ -258,9 +258,8 @@ open class AppleSequencer: NSObject {
             MusicEventIteratorPreviousEvent(iterator)
             MusicEventIteratorGetEventInfo(iterator, &eventTime, &eventType, &eventData, &eventDataSize)
             if eventType == kMusicEventType_ExtendedTempo {
-                if let data = eventData?.assumingMemoryBound(to: ExtendedTempoEvent.self) {
-                    let tempoEventPointer: UnsafePointer<ExtendedTempoEvent> = UnsafePointer(data)
-                    tempoOut = tempoEventPointer.pointee.bpm
+                if let data = eventData?.bindMemory(to: ExtendedTempoEvent.self, capacity: 1) {
```

Comment:
There's a lot of `eventData?` and could that optional be unwrapped earlier? Perhaps that could be saved for a subsequent change.